sema/commons/glob/globbery.py

```
@@ -0,0 +1,124 @@
+    and none of the globs in `excludes`.
+    @param visitor: The visitor to call for each path.
+    @param root: The root path to search under.
+    @param includes: A list of globs to include.
+    @param excludes: A list of globs to exclude.
+    @param applying: A dictionary of apply objects to pass to the visitor.
+        Keys are globs.
+    @param onlyFiles: If True, only visit files, not directories.
+    @return: A dictionary of paths to visitor results.
+        Keys are paths being visited.
+    """
+    results: dict[Path, Any] = dict()
```

Ruff (0.8.2)

Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)